File: typescript/design-patterns/facade/naive/callers-naive.ts
// The living-room "Movie Night" button:
function movieNightButton() {
  const amp = new Amplifier();
  const lights = new Lights();
  const projector = new Projector();
  const box = new StreamingBox();

  lights.dim(10);                 // ❌ caller must know all six APIs
  projector.on();
  projector.wide();
  amp.on();
  amp.setVolume(7);
  box.play("Dune");               // …and the exact ORDER
}

// The mobile app's "Play" screen repeats the SAME six calls.
// The voice assistant's "watch a movie" repeats them AGAIN.
// Change the order once → fix it in every one of them.

File: python/design-patterns/facade/naive/callers_naive.py
def movie_night_button():
    amp = Amplifier(); lights = Lights()
    projector = Projector(); box = StreamingBox()

    lights.dim(10)                 # ❌ caller must know all six APIs
    projector.on(); projector.wide()
    amp.on(); amp.set_volume(7)
    box.play("Dune")               # …and the exact ORDER

# The mobile app and the voice assistant repeat the same dance.
# Change the order once -> fix it everywhere.